apps/remix-ide/src/assets/css/themes/bootstrap-cerulean.min.css

```diff
@@ -1723,7 +1723,10 @@ pre code {
     background-clip:padding-box;
     border:1px solid #ced4da;
     border-radius:.25rem;
-    transition:border-color .15s ease-in-out,box-shadow .15s ease-in-out
+    transition:border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+    width: 50%;
+
+
```

Comment:
I would prefer if you didn't change the minified css file. any css modifications should come as css classes in a css file local to ui library

Reply:
okay i understood @joeizang